velox/exec/tests/UnnestTest.cpp

```diff
@@ -403,3 +403,53 @@ TEST_F(UnnestTest, allEmptyOrNullMaps) {
            .planNode();
   assertQueryReturnsEmptyResult(op);
 }
+
+TEST_F(UnnestTest, arrayOfRows) {
+  auto rowType1 = ROW({INTEGER(), VARCHAR()});
+  auto rowType2 = ROW({BIGINT(), DOUBLE(), VARCHAR()});
+  std::vector<std::vector<variant>> array1{
+      {
+          variant::row({1, "red"}),
+          variant::row({2, "blue"}),
+      },
+      {
+          variant::row({3, "green"}),
+          variant::row({4, "orange"}),
+          variant::row({5, "black"}),
+          variant::row({6, "white"}),
+      },
+  };
+  std::vector<std::vector<variant>> array2{
+      {
+          variant::row({static_cast<int64_t>(-1), 1.0, "cat"}),
```

Comment:
Does -1LL syntax work? It would be a bit more readable.

Reply:
Updated, thanks!